### DISCUSSION TITLE
weight_values in local_tensors in the aggregator function

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#141] [General]
In the aggregator function (e.g weighted_average_aggregation), what does local_tensors contain other than tensor_values and weight_values? and what is the difference in tensor_values and weight_values in local_tensors? How are the weight_values computed? Are the weight_values dependent on the dataset size of the collaborator?

**Comment by sarthakpati** [comment]
Weights are calculated based on the dataset size of each collaborator.
